src/kochka.com.mx/src/SocialMediaIcons.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import * as React from 'react';
+import sx from '@adeira/sx';
+import fbt from 'fbt';
+
+import Instagram from './design/svg/__generated__/Instagram';
+import Facebook from './design/svg/__generated__/Facebook';
+import socialLinks from './socialLinks';
+
+type Props = {|
+  +vertical?: boolean,
+  +color?: string,
+  +size?: number,
+|};
+
+export default function SocialMediaIcons(props: Props): React.Node {
+  return (
+    <div className={styles('icons', props.vertical && 'iconsVertical')}>
+      <div className={styles(props.vertical ? 'iconVertical' : 'icon')}>
+        <a
```

Comment:
Interesting code scanner warning...

Reply:
Damn, I thought we have this covered in our Eslint Config. Followup issue: https://github.com/adeira/universe/issues/2022